libs/design/src/atoms/form/radio/radio.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DaffRadioComponent } from './radio.component';
+
+
+
+@NgModule({
+  exports:[
+    DaffRadioComponent
+  ],
+  declarations: [
+    DaffRadioComponent
```

Comment:
I think you should declare and export the RadioSet here.

Reply:
Changed in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d